packages/2018-neighborhood-development/src/state/api-adapter.js

```diff
@@ -5,8 +5,8 @@ import axios from 'axios';
 
 const apiAdapter = (url, { start, success, error }) => () => (dispatch) => {
   dispatch(start());
-  return axios.get(url).then((res) => {
-    dispatch(success(res));
+  return axios.get(url).then(({ data }) => {
+    dispatch(success(data.results));
```

Comment:
Do we mind making this change? I noticed that we were storing the entire response in the Redux store, which seemed unnecessary.

Reply:
This is definitely better, but this might be breaking other things to do this since other neighborhood cards use this and are expecting to be getting the data out of the result. I think the better pattern is one that @DingoEatingFuzz set up in the 2018-housing-card with the fetch-adapter. If there was more time I would be cleaning up the code to use that pattern - this API adapter was really me fumbling to get something working, and now that I've worked with it more, I know there are better ways.